docs/content/docs/dev_guide/_index.md

```diff
@@ -10,7 +10,7 @@ weight: 5
 Welcome to the Open Service Mesh development guide!
 Thank you for joining us on a journey to build an SMI-native lightweight service mesh. The first of our [core principles](https://github.com/openservicemesh/osm#core-principles) is to create a system, which is "simple to understand and contribute to." We hope that you would find the source code easy to understand. If not - we invite you to help us fulfill this principle. There is no PR too small!
 
-To understand _what_ Open Service Mesh does - take it for a spin and kick the tires. Install it on your Kubernetes cluster by following [this guide](./install/manual_demo/_index.md).
+To understand _what_ Open Service Mesh does - take it for a spin and kick the tires by following [this manual demo guide](../install/manual_demo/_index.md).
```

Comment:
This resolves to https://deploy-preview-2878--osm-docs.netlify.app/docs/install/manual_demo/_index.md

It should resolve to https://deploy-preview-2878--osm-docs.netlify.app/docs/install/manual_demo/

I think you just need to remove the _index.md

Reply:
Hm I changed it but now if someone tries to navigate to it from the repo, it'll go to the directory not the guide itself: https://github.com/openservicemesh/osm/tree/main/docs/content/docs/install/manual_demo . I guess that's ok bc it's the only thing in that dir, wdyt?